ui/widgets/nuxeo-date-picker.js

```diff
@@ -130,7 +130,7 @@ import { I18nBehavior } from '../nuxeo-i18n-behavior.js';
           }
 
           vaadin-date-picker {
-            padding-bottom: 8px;
+            --lumo-space-xs: 0;
```

Comment:
What about enforcing the 48px of heigh?

Reply:
Please check https://github.com/nuxeo/nuxeo-elements/pull/301/commits/ff50db382554ee9975736861b5cd4b173ef15412#diff-a792001144b1e1c10daa172da59dcdc34c29ab1ba933c67a503eefc6e187e7dfR124